Classes/Search/SearchViewController.swift

```diff
@@ -157,6 +157,15 @@ SearchRecentHeaderSectionControllerDelegate {
     }
 
     // MARK: UISearchBarDelegate
+    
+    func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
+        guard let term = searchBar.text?.trimmingCharacters(in: .whitespacesAndNewlines),
```

Comment:
Personally not a fan of using a guard for the sole purpose of using the `else` statement - Can we change this out to either be an `if` statement or flip the guard around to check that the count is equal to 0 else return

Reply:
If you flip, you can also do `isEmpty` instead of `== 0`.